js/ui/overlay/ui.overlay.js

```diff
@@ -1292,7 +1284,7 @@ const Overlay = Widget.inherit({
         const container = this.option('container');
         let positionOf = null;
 
-        if(!container && position) {
+        if(!container) {
```

Comment:
Why do we change it?

Reply:
because _position already normalized on _init and has 'of' field